client/state/stats/lists/utils.js

```diff
@@ -47,25 +43,6 @@ export function getPeriodFormat( period, date ) {
 	}
 }
 
```

Comment:
`getWpcomBaseFilesUrl` is no longer needed because the endpoint supplies the full download URL.